server/services/caseloadService.ts

```diff
@@ -197,7 +204,7 @@ export default class CaseloadService {
       .filter(offender => !offender.nomisRecord.paroleEligibilityDate)
       .filter(offender => offender.nomisRecord.legalStatus !== 'DEAD')
       .filter(offender => !offender.nomisRecord.indeterminateSentence)
-      .filter(offender => offender.nomisRecord.confirmedReleaseDate || offender.nomisRecord.conditionalReleaseDate)
```

Comment:
Why remove this? There are legitimate cases which do not have a CRD which would not show up?

Reply:
This was the outcome of many discussions - as we covered at stand-up.